### DISCUSSION TITLE
posts/how-to-create-a-react-app-with-rust-and-wasm/

### DISCUSSION THREAD

**Discussion by giscus** [tkat0/tkat0.github.io#5] [Blog Comments]
# posts/how-to-create-a-react-app-with-rust-and-wasm/

In this article, I'll introduce followings through creating simple demo application.

https://tkat0.github.io/posts/how-to-create-a-react-app-with-rust-and-wasm/

**Comment by tardisgallifrey** [comment]
Yours is the first tutorial on Rust/React/Wasm that has worked.  Thank you and congratulations.

Now, I will spend the rest of the day modifying, simplifying, and trying to break it.

Good work.

**Reply by tardisgallifrey** [reply]
Almost forgot.  I did have to change one thing.  Only public functions (pub fn add) can be exported through WASM.

**Reply by tkat0** [reply]
Thank you for reading my article :grinning:
Yeah, you are right. I have updated the article.

**Comment by apoorv-ksharma** [comment]
Great Blog, simple and easy to understand
